base/compiler/abstractinterpretation.jl

```diff
@@ -2321,6 +2321,20 @@ function abstract_call_known(interp::AbstractInterpreter, @nospecialize(f),
     elseif f === UnionAll
         call = abstract_call_gf_by_type(interp, f, ArgInfo(nothing, Any[Const(UnionAll), Any, Any]), si, Tuple{Type{UnionAll}, Any, Any}, sv, max_methods)
         return abstract_call_unionall(interp, argtypes, call)
+    elseif f === _typeof_captured_variable && la == 2
+        t = argtypes[2]
+        if t isa Const
+            tv = t.val
+            rt = Const(has_free_typevars(tv) ? typeof(tv) : Core.Typeof(tv))
+        elseif isType(t)
```

Comment:
```suggestion
        elseif isconstType(t)
```
It would be probably fine if we use `isconstType` here?

Reply:
Sorry, I've been a bit busy this week, so didn't get to this yet. I don't really know why I thought I addressed Jameson's comment about `isType` in b64b1b3, thanks for fixing this up!